backend/src/property_handlers.py

```diff
@@ -43,12 +43,20 @@ def get_property():
                         schema: ErrorResponse
     """
     try:
-        print("sent request = " + str(request.get_json()))
+        #print("sent request = " + str(request.get_json()))
+        # data = {
+        #     "pagination" : {"results_per_page" : 1, "page" : 1}, 
+        #     "searchFields" : {"zip_code" : ["22903", "22904"], "bedrooms" : 1, "date_available" : "2020-12-01"}
+        # }
+
         payload = GetPropertyRequest().load(request.get_json())
         _property = app.services.property_service().get_property(payload)
+        print(_property)
+        print(len(_property))
         _property_response = GetPropertyResponse()
-        _property_response.pagination = PaginationResponse()
-        _property_response.results = PropertySchema().dump(_property)
+        _property_response.pagination = PaginationResponse(partial=True).dump(
+            {"TotalNumberOfResults" : len(_property)})
```

Comment:
{
                "results_per_page": 10,
                "page": 1,
                "totalNumberOfResults": len(_property)
}

it should be 'totalNumberOfResults'

Reply:
thanks!